File: ShlokM_590017851/Day_3_Question_1.c
#include <stdio.h>
#define MAX 10000  // Assumption: array values will be in the range -10000 to 10000
int main() 
{
    int nums[] = {2, 3, 4, 3, 5, 2, 3}; 
    int n = sizeof(nums) / sizeof(nums[0]);
    int k = 2;  

    int freq[2 * MAX + 1] = {0};  
    for (int i = 0; i < n; i++)
      {
        freq[nums[i] + MAX]++;
    }
    int sum = 0;
    for (int i = 0; i < 2 * MAX + 1; i++) 
    {
        if (freq[i] == k) 
        {
            sum += (i - MAX);  
        }
    }
    printf("Sum of elements that appear exactly %d times: %d\n", k, sum);
    return 0;
}

File: DSA/Angel_500119630/Day3_Question1.java
public class Day3_Question_1{
public static void main(String[] args) {
    int[] nums={1,2,2,3,3,9,9};
    int sol = sum(nums,2);
    System.out.println(sol);
    
}
 static int sum(int[] nums , int k){
    int max=0;
    for(int i:nums){
        if(i>max){
            max=i;
        }
    }
    int[] fre = new int[max +1];
    int sum = 0;
    for(int i : nums){
        fre[i]++ ;
    }
    for(int n = 0 ; n < fre.length;n++){
       
        if(fre[n]==k){
           
            sum += n ;
        }
    }
    return sum;
 }
}
